src/resolve/resolved-dns-query.h

```diff
@@ -38,7 +38,7 @@ struct DnsQuery {
          * TXT+SRV query. */
         DnsQuery *auxiliary_for;
         LIST_HEAD(DnsQuery, auxiliary_queries);
-        unsigned n_auxiliary_queries;
+        uint8_t n_auxiliary_queries;
```

Comment:
what's the benefit of this? i.e. this is placed between a native length ptr and a 32bit int. thus, on 32bit this will begin on 32bit boundary and the next field starts at 32bit, so using 32bit should be fine. on 64bit this will begin on 64bit boundary and the next field starts 32bit later, so pretty much the same story.